rest/utilities_testing.go

```diff
@@ -142,6 +142,8 @@ func (rt *RestTester) Bucket() base.Bucket {
 		rt.DatabaseConfig.AllowConflicts = &boolVal
 	}
 
+	rt.DatabaseConfig.SGReplicateEnabled = base.BoolPtr(false)
```

Comment:
I guess this is just to prevent all existing non-replication tests from having the overhead of a heartbeater?
A little counter-intuitive to reverse the default behaviour, but as long as we're aware of it when writing replicator tests, should be fine.